lib/ood_core/job/adapters/sge/helper.rb

```diff
@@ -47,6 +52,41 @@ def batch_submit_args(script, after: [], afterok: [], afternotok: [], afterany:
     args
   end
 
+  # @brief      Detect whether script content contains either -cwd or -wd
+  #
+  # @param      content  The script content
+  # 
+  # Examples:
+  #     #$-wd /home/ood/ondemand  # should match
+  #     #$  -wd /home/ood/ondemand  # should match
+  #     #$  -cwd /home/ood/ondemand  # should match
+  #     #$ -j yes -wd /home/ood/ondemand  # should match
+  #     #$ -j yes -o this-wd /home/ood/ondemand  # should NOT match
+  #       #$ -t 1-10:5 -wd /home/ood/ondemand  # should NOT match
```

Comment:
this doesn't match because there is whitespace before `#$`?

Reply:
Correct.